web-api/src/personalization/routes.rs

```diff
@@ -268,35 +273,7 @@ async fn recommendations(
         //     is_deprecated: false,
         // }
     };
-
-    let documents = if let Some(documents) = personalize_documents_by(
-        &storage,
-        &state.coi,
-        &user_id,
-        &state.config.personalization,
-        PersonalizeBy::KnnSearch {
-            count,
-            filter: filter.as_ref(),
-        },
-        Utc::now(),
-        include_properties,
-        include_snippet,
-    )
-    .await?
-    {
-        Either::Left(Json(PersonalizedDocumentsResponse {
-            documents: documents.into_iter().map_into().collect(),
-        }))
-    } else {
-        Either::Right((
-            Json(PersonalizedDocumentsError::NotEnoughInteractions),
-            StatusCode::CONFLICT,
-        ))
-    };
-
-    Ok(deprecate!(if is_deprecated {
-        documents
-    }))
+    _recommendations(state, request, storage).await
```

Comment:
:art: Leading `_` has a special meaning in rust it means: "don't warn if this thing is not used".

Note that this isn't the same as dead code. I.e. it's mainly used for variables where you assign a value which has some side-effect when being dropped but doesn't need to be accessed anymore (e.g. release lock on end of scope). 

So it's better to rename this to e.g.  `recommendations_inner`, `recommendations_` or similar.

Reply:
done